src/www/services_dyndns_edit.php

```diff
@@ -291,141 +291,141 @@ function _onTypeChange(type){
                     </td>
                   </tr>
                   <tr id="_hostnametr">
-                    <td><a id="help_for_host" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Hostname");?></td>
+                    <td><a id="help_for_host" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext("Hostname") ?></td>
                     <td>
-                      <input name="host" type="text" id="host" value="<?=$pconfig['host'];?>" />
+                      <input name="host" type="text" id="host" value="<?= $pconfig['host'] ?>" />
                       <div class="hidden" for="help_for_host">
-                        <span class="text-danger"><strong><?=gettext("Note:");?><br /></strong></span>
-                        <?=gettext("Enter the complete host/domain name.  example:  myhost.dyndns.org");?><br />
-                        <?=gettext("For he.net tunnelbroker, enter your tunnel ID");?>
+                        <span class="text-danger"><strong><?= gettext("Note:") ?><br /></strong></span>
```

Comment:
Please let us get rid of all the iffy usage of Note: Warning: Whatnot: and the weird markup thereof. Useful help message and help message only. :)